index.js

```diff
@@ -21,8 +23,9 @@ function accesorString(value) {
 
 module.exports = function() {};
 module.exports.pitch = function(remainingRequest) {
+	var newRequestPath = "." + path.sep + path.basename(remainingRequest);
```

Comment:
Yes, there should be a short comment here with a high level explanation then reference the pull request #